src/test/test.rs

```diff
@@ -21,6 +21,14 @@ mod tests {
         }
     }
     #[test]
+    // Test loading of config file from directory with custom DNS for containers
+    fn test_loading_config_file_with_dns_servers() {
+        match config::parse_configs("src/test/config/podman_custom_dns_servers") {
+            Ok(_) => {}
+            Err(e) => panic!("{}", e),
+        }
```

Comment:
why not just call unwrap()?

Reply:
fixed for this test, will open different PR for older tests.